scripts/setup-env.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable consistent-return, no-console */
+
+const fs = require('fs');
+const { exec } = require('child_process');
+const path = require('path');
+
+function updateConfig(directory) {
+  if (process.env.ISSUER === undefined || process.env.CLIENT_ID === undefined) {
+    console.log('[ERROR] Please set the ISSUER and CLIENT_ID Environment variables');
+    return;
+  }
+
+  const file = path.join(__dirname, '/../', directory, '/src/app/.samples.config.ts');
+  const data = fs.readFileSync(file, 'utf8');
+  let result = data.replace(/{clientId}/g, process.env.CLIENT_ID);
+  result = result.replace(/https:\/\/{yourOktaDomain}.com\/oauth2\/default/g, process.env.ISSUER);
+  fs.writeFileSync(file, result, 'utf8');
+}
+
+function cloneRepository(repository, directory) {
+  const dir = path.join(__dirname, '/../', directory);
```

Comment:
I'm used to only using `'..'` instead of `'/../'` when joining paths, are the slashes needed?

Reply:
That works. `/../` not needed. will update